Dangerfile

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Runs rails_best_practices on modified and added files in the PR
+rails_best_practices.lint
+
+# Runs Rubocop and submit comments on modified and added files
+rubocop.lint(inline_comment: true, force_exclusion: true)
+
+# Runs Brakeman on modified and added files
+brakeman.run
+
+# Runs Reek on modified and added files
+reek.lint
```

Comment:
We still keep using `reek`, should we convert the `reek` config file from `config.reek` to `.reek.yml`? Then we can use it as the default configuration

Reply:
Updated!